src/eventer/eventer_impl.c

```diff
@@ -236,12 +258,20 @@ int eventer_impl_propset(const char *key, const char *value) {
     return 0;
   }
   if(!strncasecmp(key, "loop_", strlen("loop_"))) {
+    char *nv = alloca(strlen(value)+1), *tok;
+    memcpy(nv, value, strlen(value)+1);
     const char *name = key + strlen("loop_");
     if(strlen(name) == 0) return -1;
 
-    int requested = atoi(value);
+    ADVTOK; /* concurrency */
+    int requested = tok ? atoi(tok) : 0;
+    ADVTOK;
+    double hb_timeout = tok ? atof(tok) : 0;
```

Comment:
should we have error checking here? `strtod`, and verify that the string ends with `null`, and that `hb_timeout >= 0`?